.github/workflows/test.yml

```diff
@@ -21,14 +21,15 @@ jobs:
   workspace_deluxe_tests:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         include:
-          - java: '8'
+          - java: '11'
             mongo: 'mongodb-linux-x86_64-3.6.23'
             minio: '2019-05-23T00-29-34Z'
-            wired_tiger: 'false'
+            wired_tiger: 'true'
             ant_test: 'test_quick_coverage'
-          - java: '11'
+          - java: '17'
```

Comment:
these tests fail - this section could be considered aspirational

Reply:
Removed, I don't want the tests to fail every time we PR. Maybe just have another java 11 test with wired tiger off and mongo version 3.6.13 which is in prod. The test above can be "cutting edge" mongo weeps histrionically

Reply:
:+1: